UI/Enums/OrganisationTypeGrouping.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.ComponentModel;
+
+namespace UI.Enums
```

Comment:
The enum file should be in the `Domain/Enums` folder instead of here.

Reply:
Agreed, think I missed this because I just copied the existing TuitionType filter which has it there.  But you'll notice in https://github.com/DFE-Digital/find-a-tuition-partner/pull/216 that I have already moved TuitionType to Domain/Enums.  So I'll move OrganisationTypeGrouping and TuitionType and others will be updated as part of PR 216

Reply:
Okay, that's fine.